src/test/java/kostka/moviecatalog/service/CacheServiceTest.java

```diff
@@ -62,6 +66,8 @@ public void get5LatestMoviesFromRedisCacheTest() throws JsonProcessingException
         List<MovieListDto> result = cacheService.getMoviesFromCacheWithKey(LATEST_MOVIES_KEY);
 
         assertThat(result.size()).isEqualTo(2);
+        assertThat(result).extracting(
```

Comment:
A proč tady není inAnyOrder?

Reply:
tady je to proto, ze to ma byt serazeny seznam :)
ale jelikoz se ten list mokuje, asi to nema smysl :)